src/test/java/org/minions/devfund/benjamin/MovieTest.java

```diff
@@ -0,0 +1,53 @@
+package org.minions.devfund.benjamin;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * class to tes movie class.
+ */
+public class MovieTest {
+    /**
+     * Test set name function.
+     */
+    @Test
+    public void testSetName() {
+        Movie movie = new Movie();
+        String movieName = "BATMAN";
+        movie.setName(movieName);
+        assertEquals(movieName, movie.getName());
+    }
+
+    /**
+     * Test add actors function.
+     */
+    @Test
+    public void testAddActors() {
+        Movie movie = new Movie();
+        String movieName = "BATMAN";
+        movie.setName(movieName);
+        movie.addActors(new String[]{"Crhistian Bale", "Morgan Freeman"});
+        assertEquals(2, movie.getSize());
+    }
+
+    /**
+     * Test get movie name function.
+     */
+    @Test
+    public void testGetName() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "testSetName" on line 13. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)